### ISSUE TITLE
Crash when external links are enabled

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#1740]
```
11-03 09:32:05.683 7070-7188/com.nextcloud.client E/AndroidRuntime: FATAL EXCEPTION: Thread-15
                                                                    Process: com.nextcloud.client, PID: 7070
                                                                    android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
                                                                        at android.view.ViewRootImpl.checkThread(ViewRootImpl.java:6855)
                                                                        at android.view.ViewRootImpl.requestLayout(ViewRootImpl.java:1040)
                                                                        at android.view.View.requestLayout(View.java:19658)
                                                                        at android.view.View.requestLayout(View.java:19658)
                                                                        at android.view.View.requestLayout(View.java:19658)
                                                                        at android.view.View.requestLayout(View.java:19658)
                                                                        at android.view.View.requestLayout(View.java:19658)
                                                                        at android.view.View.requestLayout(View.java:19658)
                                                                        at android.support.v4.widget.DrawerLayout.requestLayout(DrawerLayout.java:1265)
                                                                        at android.view.View.requestLayout(View.java:19658)
                                                                        at android.view.View.requestLayout(View.java:19658)
                                                                        at android.support.v7.widget.RecyclerView.requestLayout(RecyclerView.java:3875)
                                                                        at android.support.v7.widget.RecyclerView$RecyclerViewDataObserver.onChanged(RecyclerView.java:4945)
                                                                        at android.support.v7.widget.RecyclerView$AdapterDataObservable.notifyChanged(RecyclerView.java:11357)
                                                                        at android.support.v7.widget.RecyclerView$Adapter.notifyDataSetChanged(RecyclerView.java:6634)
                                                                        at android.support.design.internal.NavigationMenuPresenter$NavigationMenuAdapter.update(NavigationMenuPresenter.java:461)
                                                                        at android.support.design.internal.NavigationMenuPresenter.updateMenuView(NavigationMenuPresenter.java:117)
                                                                        at android.support.v7.view.menu.MenuBuilder.dispatchPresenterUpdate(MenuBuilder.java:292)
                                                                        at android.support.v7.view.menu.MenuBuilder.onItemsChanged(MenuBuilder.java:1047)
                                                                        at android.support.v7.view.menu.MenuBuilder.addInternal(MenuBuilder.java:452)
                                                                        at android.support.v7.view.menu.MenuBuilder.add(MenuBuilder.java:475)
                                                                        at com.owncloud.android.ui.activity.DrawerActivity.updateExternalLinksInDrawer(DrawerActivity.java:1015)
                                                                        at com.owncloud.android.ui.activity.DrawerActivity$9.run(DrawerActivity.java:1357)
                                                                        at java.lang.Thread.run(Thread.java:761)
```

- have external links configured in your server
- fresh install of the app

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky is this fixed via #1795 ?

**Comment by tobiasKaminsky** [comment]
Correct.
